stable/stolon/values.yaml

```diff
@@ -74,6 +74,17 @@ clusterSpec: {}
   # sleepInterval: 1s
   # maxStandbys: 5
 
+## Enable support ssl into postgres, you must specify the certs.
+## ref: https://www.postgresql.org/docs/10/ssl-tcp.html
+##
+tls:
+  enabled: false
+  rootCa: |-
```

Comment:
I wonder, should it be always up to the user to create a secret, to not have private key in the values.yaml

Reply:
Hello, thanks for your comments. The user can choose, secrets can be created before and specify the name of the secret using

`` suggestion
createdSecret: existing-secret-name-a-postgresql
``

or type the certificate in values.yaml.
I want to assume that the user encrypts sensible information in the values.yaml file, for example, using helm secrets